src/libraries/System.Data.Common/src/System/Data/SqlDbType.cs

```diff
@@ -38,5 +38,6 @@ public enum SqlDbType
         Time = 32,
         DateTime2 = 33,
         DateTimeOffset = 34,
+        Json = 35,
```

Comment:
Nit: I realize none of the other values have XML comments, but in general we use XML comments to seed the docs. I'd suggest adding one here. (For bonus points but not required, you could make the whole type consistent by copying the XML comments from the docs for all the other values. :)

Reply:
@stephentoub I will add the XML docs for all the types here. The only trouble I have is with linking of these XML docs with the other types. The doc pages show that the comments of each of these enums link to types like `SqlXml` or `Array` etc. 
Not sure how that works. But I agree that this file could benefit from atleast adding the XML docs. I will send an update to the PR.

Reply:
> I will add the XML docs for all the types here.

Ok, thanks :)

> Not sure how that works.

e.g. `<see cref="Array" />`

Reply:
Great. That's not too bad then. The existing XML file in the docs repo, already has the right format, which can be brought over. 
e.g. 
```
       <summary>An XML value. Obtain the XML as a string using the <see cref="M:System.Data.SqlClient.SqlDataReader.GetValue(System.Int32)" /> method or <see cref="P:System.Data.SqlTypes.SqlXml.Value" /> property, or as an <see cref="T:System.Xml.XmlReader" /> by calling the <see cref="M:System.Data.SqlTypes.SqlXml.CreateReader" /> method.</summary>

```